examples/google.events.cloud.audit.v1.md

```diff
@@ -0,0 +1,34 @@
+## google.events.cloud.audit.v1
```

Comment:
Any reason for having this as a Markdown file rather than just a shell script? Shell scripts still render nicely in GitHub, and that way anyone who has cloned the repo could just call the shell script directly rather than copying it into a new file.

Reply:
I'd like to have a doc to be able to put documentation about the event.

This product really really needs docs and explanation. We can figure out a better automated testing strategy later, but I think at minimum, we should create a great copy-paste experience, which this provides.